evap/evaluation/migrations/0100_clear_evaluation_names.py

```diff
@@ -22,7 +22,7 @@ def name_evaluations(apps, _schema_editor):
             evaluation.name_en = course.name_en
             evaluation.save()
         else:
-            for i in range(0, course.evaluations.count()):
+            for i in range(course.evaluations.count()):
                 evaluation = Evaluation.objects.get(pk=course.evaluations.all()[i].pk)
```

Comment:
Never expected to see such cursed code in EvaP. Probably doesn't really matter since its a rather old migration.